### PR TITLE
Config firebase

### PR DESCRIPTION
**Pull request by hoangdat**
- [x] config Android to work with `firebase`
- [x] config iOS to work with `firebase`
  - https://firebase.flutter.dev/docs/messaging/apple-integration/

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -130,6 +130,7 @@ dependencies:
   # fk_user_agent
   fk_user_agent: 2.1.0
 
+  firebase_analytics: 9.1.2
```
Comment (hoangdat) [inline pubspec.yaml:133]: why we should add analytic also?

Comment (dab246) [reply pubspec.yaml:133]: I think there should be so we can see the log of events and actions from firebase. This firebase library is also recommended.

**Comment by dab246** [comment]
@hoangdat  Please test it on real iOS device

File: ios/Runner/Info.plist [hunk 2]
```diff
@@ -22,16 +22,21 @@
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSContactsUsageDescription</key>
+	<string>To send email with these contacts</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>To save photo to library, we need permission to access your photo library.</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>To select and upload photo, we need permission to access your photo library.</string>
+	<key>UIBackgroundModes</key>
```
Comment (hoangdat) [inline ios/Runner/Info.plist:31]: Did you forget to enable `Push notification`?

Comment (dab246) [reply ios/Runner/Info.plist:31]: @@ it's enabled but it doesn't need to be declared here

**Pull request merged**